ui/page/components/account_selector.go

```diff
@@ -94,9 +94,9 @@ func (as *AccountSelector) Handle() {
 }
 
 // SelectFirstWalletValidAccount selects the first valid account from the
-// first wallet in the SortedWalletList, skipping any account whose ID is == filter.
+// first wallet in the SortedWalletList, skipping any account whose number is in filterAccounts slice.
```

Comment:
From your comment, it say skip any account whose ID is found... I believe this should mea n not displayed among options or can be selected.

I can however still see and select the default wallet.
I think the best solution is not to have it among the options to be selected. this reduces chances of errors.

<img width="498" alt="image" src="https://user-images.githubusercontent.com/27733432/165468247-418048b5-633f-4e62-8b18-59ae9c603d73.png">

i see that the set up button is greyed out yeah.. lets just give the users the available options only.